files/conf/kubelet-docker-config.json

```
@@ -0,0 +1,7 @@
+{
+    "auths": {
+        "https://index.docker.io/v1/": {
+            "auth": {{ .DockerhubToken }}
```

Shouldn't be quoted?

```suggestion
            "auth": "{{ .DockerhubToken }}"
```